llvm/lib/Transforms/Vectorize/SLPVectorizer.cpp

```diff
@@ -23325,6 +23329,8 @@ class HorizontalReduction {
     case RecurKind::FMulAdd:
     case RecurKind::AnyOf:
     case RecurKind::FindLastIV:
+    case RecurKind::FindFirstIVUMin:
+    case RecurKind::FindFirstIVSMin:
```

Comment:
Why is this asymmetric? Wouldn't it be nicer to first separate out FindLastIV into FindLastIVSMax/UMax?